addOns/ascanrulesAlpha/src/test/java/org/zaproxy/zap/extension/ascanrulesAlpha/ForbiddenBypassScanRuleUnitTest.java

```diff
@@ -74,22 +89,22 @@ void shouldNotAlertIfAllRequestsAreForbidden() throws Exception {
         rule.scan();
         // Then
         assertThat(alertsRaised, hasSize(0));
-        assertEquals(23, httpMessagesSent.size());
+        assertEquals(25, httpMessagesSent.size());
```

Comment:
Not sure worth checking the exact number.

Reply:
But we should check that something was sent though.

Reply:
Now checks messages sent > 20